scrapers/Waybig.yml

```diff
@@ -0,0 +1,67 @@
+name: Waybig
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - waybig.com/video/
+    scraper: sceneScraper
+performerByURL:
+  - action: scrapeXPath
+    url:
+      - waybig.com/pornstars/
+    scraper: performerScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //meta[@name="twitter:title"]/@content
+      Date:
+        selector: //i[@class="far fa-calendar"]/following-sibling::text()
+        postProcess:
+          - replace:
+              - regex: ^(\d{4}-\d{2}-\d{2}).+
+                with: $1
+          - parseDate: 2006-01-02
+      Details:
+        selector: //div[@class="expand-inner"]
+      Tags:
+        Name: //div[contains(text(),"Tags:")]/following-sibling::a
+      Performers:
+        Name:
+          selector: //div[@class="models-box"]/a
+      Image:
+        selector: //video/@poster
+      Studio:
+        Name:
+          fixed: Waybig
+  performerScraper:
+    performer:
+      Name: //div[@class="title-col -normal col"]/h2
+      Gender:
+        fixed: Male
+      Measurements:
+        selector: //span[contains(text(),"Cock size:")]/following-sibling::text()
```

Comment:
```yaml
        selector: //span[contains(text(),"Cock size:")]/following-sibling::text()
        postProcess:
          - replace:
              - regex: ^
                with: "0''"
          - feetToCm: true
```
a small hack to get the size in cm (accuracy is not that good though since the decimal point will be ignored)